modules/osd-monitoring-understanding-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-understanding-the-monitoring-stack.adoc
+
+[id="understanding-the-monitoring-stack_{context}"]
+= Understanding the monitoring stack
+
+The {product-title} monitoring stack is based on the link:https://prometheus.io/[Prometheus] open source project and its wider ecosystem. The monitoring stack includes the following:
+
+* *Default platform monitoring components*. A set of platform monitoring components are installed in the `openshift-monitoring` project by default during an {product-title} installation. This provides monitoring for core {product-title}. The default monitoring stack also enables remote health monitoring for clusters. Critical metrics are collected from all of the workloads in every namespace, such as CPU and memory, and are made available for your use.
```

Comment:
"during an {product-title} installation" renders in the build as "during an Red Hat OpenShift Service on AWS installation". You could change this to "during a {product-title} installation".

Reply:
Done.